test_cromwell.sh

```diff
@@ -86,13 +86,17 @@ if [[ -n ${CROMWELL_BRANCH} ]]; then
     git checkout "${CROMWELL_BRANCH}"
     git pull
     echo "Building Cromwell"
-    sbt clean assembly >> ../${ASSEMBLY_LOG} 2>&1
+    sbt clean assembly >> "${ASSEMBLY_LOG}" 2>&1
     cd ..
-    CROMWELL_JAR="cromwell/target/scala-2.11/cromwell-*.jar"
+    CROMWELL_JAR=`find "${RUN_DIR}"/cromwell/target/scala-2.11 -name "cromwell-*.jar"`
```

Comment:
Heh i saw that in the bash linter but at that point i was so frustrated i didn't want to rock the boat. I'll try it and make sure it doesn't break anything